### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by theecolya**
## Description

I added myself and fixed the weird thing that was happening with the other names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,8 @@ You can find the deployed project at [CoderHeroes](https://a.coderheroes.dev/log
...
- |
+| [<img src="https://avatars.githubusercontent.com/u/83845399?v=4" width = "200" />](https://github.com/waylonturbes)                            | [<img src="https://avatars.githubusercontent.com/u/87030206?v=4" width = "200" />](https://github.com/krisf451)                              | [<img src="https://avatars.githubusercontent.com/u/56285755?v=4" width = "200" />](https://github.com/seanmp3)                                         | [<img src="https://avatars.githubusercontent.com/u/98560786?v=4" width="200"> ](https://github.com/theecolya) |
+| [Waylon Turbes](https://github.com/waylonturbes) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/waylonturbes)     | [Kristian Fulkerson](https://github.com/krisf451) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/krisf451)      | [Sean Mahan](https://github.com/seanmp3) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/seanmp3)                          |[Colya Foxworth](https://github.com/theecolya) <img src="https://github.com/favicon.ico" width="15">
```
Comment (MaxT6) [inline README.md:27]: I am not sure if this matters but do you need a bracket in front of the last <img tag?

**Review by MaxT6 — APPROVED** [review]
Looks good! I have one question that I noted but it is probably inconsequential.

File: README.md (hunk 1 above)
Comment (mcfrr3) [reply README.md:27]: @MaxT6 I think that's what makes it a link in markdown

Comment (mcfrr3) [reply README.md:27]: I went down a bit too far down a rabbit hole but something [like this](https://lukasznojek.com/blog/2020/03/generate-markdown-files/) might be interesting as away to dynamically generate these tables

**Comment by Austin-T-Johnson** [comment]
Good job taking it upon yourself to fix an issue that you noticed in the README. Perhaps you could have been a bit more descriptive in the description of this PR to tell us what the weird thing was!

**Review by andrewgomez1992 — APPROVED** [review]
Great job staying active and actually going through the README.md, instead of the 'weird thing' in the description, we would love to know the exact issue! Overall great job :)

**Review by Epeng1994 — APPROVED** [review]
Great job on noticing this issue!

**Review by macrowell70 — APPROVED** [review]
Good idea adding yourself to the contributors page, and fixing a bug in the process.

**Pull request merged**